modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java

```diff
@@ -194,6 +203,32 @@ protected String getFileEntryURL(
 		return html.escape(sb.toString());
 	}
 
+	protected String getItemSelectorURL(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext,
+		long folderId, HttpServletRequest httpServletRequest) {
```

Comment:
@carolmariaabb, I think it would be better to add the `folderId` as the first parameter once it's a `id` and considering the previous reviews of Brian, what do you think?

Reply:
Hi @natocesarrego. I did it that way because of this [change](https://github.com/brianchandotcom/liferay-portal/commit/e15357135d1b8b08765c078b953d488f95051f9e) made by Brian in the previous PR.

Reply:
Ok. Therefore, I can conclude that move `id` parameters to the beginning of the list of parameters should only be done for methods on `*Service.java` and `*LocalService.java`. Thanks for your clarification, @carolmariaabb.